pkg/reconciler/cache/cachedresources/replication/replication_reconcile.go

```diff
@@ -237,6 +211,54 @@ func (c *Controller) reconcile(ctx context.Context, gvrKey string) error {
 			return c.kcpCacheClient.Cluster(cluster.Path()).CacheV1alpha1().CachedObjects().Delete(ctx, cachedObjName, metav1.DeleteOptions{})
 		},
 	}
+	// r.getLocalCopy is defined separately because it calls r.getGlobalCopy internally.
+	r.getLocalCopy = func(cluster logicalcluster.Name, namespace, name string) (*unstructured.Unstructured, error) {
```

Comment:
Name now does now make sense. Can we rename to something better as it calls both now?